.github/workflows/link-validator.yml

```diff
@@ -12,15 +12,10 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
-          # See https://github.com/actions/checkout/issues/299#issuecomment-677674415
```

Comment:
Resolved in https://github.com/actions/checkout/issues/237#issuecomment-667158637